src/bank_editor.cpp

```diff
@@ -760,6 +760,30 @@ void BankEditor::on_actionReMeasure_triggered()
     }
 }
 
+void BankEditor::on_actionReMeasureOne_triggered()
+{
+    FmBank::Instrument *inst = m_curInst;
+    FmBank::Instrument *instBackup = m_curInstBackup;
+    if(!inst)
+    {
+        QMessageBox::information(this,
+                                 tr("Nothing to measure"),
+                                 tr("No selected instrument to measure. Please select an instrument first!"));
+        return;
+    }
+
+    FmBank::Instrument workInst = *inst;
+    if(workInst.is_blank)
```

Comment:
I think, this flag is must be set by measurer, and possibly, new-created instrument from scratch will be skipped. The blank flag is installing by measurer when 6 windows of silence has detected

Reply:
I dislike how measurer fixes windows to a fixed size in samples.
Not only it's too small to contain full periods of some frequencies, it makes result depending on sample rate.
I made the new method distinguishing the window size and step size, and I'm sizing the window according to fixed time in seconds.

Reply:
That yeah, but having the "is_blank" flag to block the measurement will give no way to measure the instrument made from scratch. 
Instead of "is_blank" check, do:
```cpp
FmBank::Instrument blank = FmBank::emptyInst();
ins.is_blank = false; //this must be to don't make the false difference.
if(memcmp(&ins, &blank, sizeof(FmBank::Instrument)) == 0)
    return;
```